app/views/stages/_fields.html.erb

```diff
@@ -28,7 +28,7 @@
       <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
       <div class="col-lg-4">
         <%= form.collection_select(
```

Comment:
Why wouldn't I want to add the `includes(:environment)`?  Is that something that's now handled by that gem you added?